site/en/configure/build-performance-metrics.md

```diff
@@ -0,0 +1,97 @@
+Project: /_project.yaml
+Book: /_book.yaml
+
+# Extracting build performance metrics
+
+{% include "_buttons.html" %}
+
+Probably every Bazel user has experienced builds that were slow or slower than
+anticipated. Improving the performance of individual builds has particular value
+for targets with significant impact, such as:
+
+1. Core developer targets that are frequently iterated on and (re)built.
+
+2. Common libraries widely depended upon by other targets.
+
+3. A representative target from a class of targets (e.g. custom rules),
+  diagnosing and fixing issues in one build might help to resolve issues at the
+  larger scale.
+
+An important step to improving the performance of builds is to understand where
+resources are spent. This page lists different metrics you can collect.
+[Breaking down build performance](/build-performance-breakdown) showcases
```

Comment:
This reference needs a `/configure` prefix

Reply:
Done.